vibe/src/main/java/com/vibeapp/vibe/models/ExtendedView.java

```diff
@@ -0,0 +1,100 @@
+package com.vibeapp.vibe.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name="profile")
+
+public class ExtendedView
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int uId;
```

Comment:
weird naming here. going to cause issues with findBy methods in the JPA repository